### ISSUE TITLE
Say all doesn't work properly when synthesyzer is set to no speech

### ISSUE THREAD

**Issue by ondrosik** [NVDARemote/NVDARemote#217]
Steps to reproduce:
- set synthesyzer to none or speech mode to off and try reading something by say all
- Cursor will move just one line forward.

This is causing issues with NVDA Remote when you disable speech on remote computer. Say all then doesn't work and reads just one line then stops. Expected behavior is that also in this situation say all will read the document. This was usefull in situations where you set synthesyzer to "none" on remote computer but want to read something on remote computer. This worked until NVDA 2019.3. I am not sure if this should be fixed in NVDA or addon, NVDA issue is [here](https://github.com/nvaccess/nvda/issues/10797)
